### DISCUSSION TITLE
How to implement autoinhibition

### DISCUSSION THREAD

**Discussion by AyaSaleh90** [Open-Systems-Pharmacology/Forum#1185] [Questions & Problems]
Dear OSP community,

I’m developing a coupled PBPK model for a compound & its major metabolites based on in vitro analyses generated in our research group combined with clinical data. The in vitro analyses showed that the parent is metabolised via three CYP enzymes. Still, it inhibits 2 of those CYP enzymes with a competitive mechanism and the other one with a non-competitive mechanism. While the metabolites were not metabolised via those enzymes, but they inhibited them as the parent (they inhibit their own formation).  

I have managed to apply the inhibition parameters of one of the metabolites to the parent. Still, my question is how to implement the autoinhibition processes of the parent compound? Although I checked the documentation of PK-Sim, and it was mentioned that “autoinhibition cannot be set up because measured Ki values will already be altered due to autoinhibition”, So has anyone of you had a similar experience and could they point me towards a possible solution?
Please let me know if anything is unclear or more information should be provided.

Thanks in advance,
Aya

**Comment by moPellowe** [comment]
Hello Aya,
Just to clarify, have you already looked at adding an inhibition process in the compound (as in the figure below)?

![image](https://user-images.githubusercontent.com/57540473/160617019-efa9708d-9741-40dc-a860-bdf320b970d6.png)

**Comment by AyaSaleh90** [comment]
Hi Moriah, 

Thanks for your suggestion and yes, I have applied this way for the parent, but it did not have an effect on the predicted concentration-time profile for the parent. 
and it worked only when I defined those inhibition parameters of the metabolites on the parent's elimination processes. but it doesn't work for the parent on its own metabolism

**Reply by StephanSchaller** [reply]
Hi @AyaSaleh90, this is already addressed in https://github.com/Open-Systems-Pharmacology/docs/issues/112 , right? So there is no "need" to add this auto-inhibition as Km includes it.

Using the competitive inhibition, the addition of auto-inhibition (Ki = Km) would result in the enzyme kinetics of: `Vmax*S/(Km+2*S)` with S being the concentration of substrate=inhibitor. depending on the Km and substrate concentrations (i.e., are you still in the linear range of kinetics) you would likely not see an effect also if the inhibition is added.

**Reply by AyaSaleh90** [reply]
Hi @StephanSchaller, yes this was the same problem I addressed in Open-Systems-Pharmacology/docs#112. 
Thanks a lot for clarifying the part of that Km already includes the Ki, I got this point.

But I have one more question, what exactly do you mean by a linear range of kinetics? 
For example, the drug of question in the very low doses has linear kinetics which is observed and captured by the PBPK model using the Km from the in-vitro analyses. While upon increasing the dose the CL of the drug decreases over time and nonlinearity become more pronounced, and that's why I hypothesized that inhibition is not characterised in the right way.

**Reply by StephanSchaller** [reply]
with *linear range* I meant the case if substrate *S* is << than *Km*
